schema/3pse/file/applications/espresso/7.2/pw.x/hubbard.json

```diff
@@ -27,7 +27,7 @@
                                 "description": "https://www.quantum-espresso.org/Doc/INPUT_PW.html#idm1764",
                                 "additionalProperties": false,
                                 "properties": {
-                                    "U": {
+                                    "hubbard_param": {
```

Comment:
I see that the enums for `hubbard_param` and `hubbard_val` change, but I think we can isolate `label` and `manifold` to a reusable schema.

Reply:
Let's just reuse the `scalar` schema - it has `name` and `value` - same for the below

Reply:
considering label and manifold are only used in this schema, and has two different variations, I think it is not worth separating into a new schema, until they are needed elsewhere; at the moment this seems unlikely. If they are used in other espresso executables, we can pull them out into a reusable sub-schema at that time.
scalar schema should not be used since then we lose descriptions of the fields which I think is very important for these template to be maintainable.